spec/rspec/matchers/dsl_spec.rb

```diff
@@ -708,11 +708,8 @@ def foo
             end
           end
 
-          begin
-            aggregate_failures do
-              expect(1).to rescue_failure
-            end
-          rescue RSpec::Expectations::ExpectationNotMetError
+          aggregate_failures do
+            expect(1).to rescue_failure
```

Comment:
You can disable the cop here, and restore this, its to prevent a failure causing the suite problems

Reply:
Got it, makes total sense.